src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/AddressEndpoint.java

```diff
@@ -43,8 +45,21 @@ public AddressEndpoint(final AddressService addressService) {
    */
   @RequestMapping(value = "", method = RequestMethod.GET)
   public AddressQueryResponseDTO getAddressesBySearchQuery(
-      @Valid AddressQueryRequestDTO addressQueryRequest) {
+      @Valid AddressQueryRequestDTO addressQueryRequest) throws CTPException {
     log.with("requestParams", addressQueryRequest).info("Entering GET getAddressesBySearchQuery");
+
+    String addressQueryInput =
+        addressQueryRequest.getInput().trim().replaceAll("'", "").replaceAll(",", "");
+
+    if (addressQueryInput.length() < 5) {
+      throw new CTPException(
+          Fault.BAD_REQUEST,
+          "Address query requires 5 or more characters, "
+              + "not including single quotes, commas or leading/trailing whitespace");
+    }
+
+    addressQueryRequest.setInput(addressQueryInput);
```

Comment:
Why do we replace the input with the input used to validate ?    We know we have valid input, so can't we leave as-is ? eg the comma might be relevant to search matching.

Reply:
Hmmm. Good point - AI will do the same? validation when they receive the query, so as long as it passes ours - we will need to retest that with keeping the input and sending to AI once passed our validation.

Reply:
But we will have to retest it in dev Rayson. We think we understand AIs requirement, but this is AI we are talking about...

Reply:
I have removed line 61, can test whenever